pkg/segment/results/segresults/segresults.go

```diff
@@ -368,6 +368,26 @@ func (sr *SearchResults) UpdateSegmentStats(sstMap map[string]*structs.SegStats,
 			}
 			sort.Strings(uniqueStrings)
 			strVal := strings.Join(uniqueStrings, "&nbsp")
+			sr.segStatsResults.measureResults[measureAgg.String()] = utils.CValueEnclosure{
+				Dtype: utils.SS_DT_STRING,
+				CVal:  strVal,
+			}
+			continue
+		case utils.List:
+			strList := make([]string, 0, 100)
+
+			if currSst != nil && currSst.StringStats != nil && currSst.StringStats.StrList != nil {
+				strList = utils.AppendWithLimit(strList, currSst.StringStats.StrList, 100)
+			}
+
+			if sr.runningSegStat[idx] != nil && sr.runningSegStat[idx].StringStats != nil {
+				strList = utils.AppendWithLimit(strList, sr.runningSegStat[idx].StringStats.StrList, 100)
+			}
+
+			sort.Strings(strList)
+
+			strVal := strings.Join(strList, " ")
+
 			sr.segStatsResults.measureResults[measureAgg.String()] = utils.CValueEnclosure{
 				Dtype: utils.SS_DT_STRING,
 				CVal:  strVal,
```

Comment:
Is this how we store a multivalue value? I thought we changed it so we store a list of strings

Reply:
Addressed this in the latest commit.